nessie/golden/lakectl_fs_second_upload.golden

```diff
@@ -0,0 +1,7 @@
+Path: upload_file.txt.1
+Modified Time: <DATE> <TIME> <TZ>
+Size: 20 bytes
+Human Size: 20 B
+Physical Address: ${STORAGE}/<OBJECT_KEY>
+Checksum: bcd52aa37b0c6151513579d6f21c4414
+Content-Type: application/octet-stream
```

Comment:
Looks like you were forced to create another upload output golden file, due to lakectl tests limitation
Can you please open an issue on lakectl-testing so we can parametrized the path, size and checksum, and reuse the golden file?
Thanks

Reply:
Sure